src/core/server/elasticsearch/client/configure_client.test.ts

```diff
@@ -139,6 +139,11 @@ describe('configureClient', () => {
       const response = createApiResponse({
         statusCode: 400,
         headers: {},
+        params: {
+          method: 'GET',
+          path: '/foo',
```

Comment:
super nit: it's a little confusing that the path here doesn't match the one in the `reason` which is `/_path`